gulp-tasks/build-node-packages.js

```diff
@@ -1,9 +1,34 @@
 const gulp = require('gulp');
+const fs = require('fs-extra');
+const path = require('path');
 
 const buildNodePackage = require('./utils/build-node-package');
 const packageRunnner = require('./utils/package-runner');
 
+// This is needed for workbox-build
+gulp.task('build-node-packages:update-cdn-details', function() {
```

Comment:
Just checking on the release flow when this task is included:

1. It gets run as part of the `build-node-packages` task.
1. It will potentially update `packages/workbox-build/src/cdn-details.json`.
1. If it does update the file, the modification won't be committed to `git`.

Could that lead to problems when `build-node-packages` is being run as part of a release? I.e. will the release only publish whatever is currently committed to `git`, and not include the pending change?

Reply:
When lerna is publishing it'll call `npm run version` which is called *after* the version has been bumped but before anything is committed.

workbox-build will build because of the `npm run version` script calling `npm run build` This means that the cdn version will be updated in workbox-build before it's commited.

Realistically, we only need to build after the version bump, but we run build in prepare because that is what lerna's bootstrapping will call.